code/AssetLib/glTF2/glTF2Asset.inl

```diff
@@ -1628,6 +1628,22 @@ inline void Node::Read(Value &obj, Asset &r) {
                 }
             }
         }
+
+        // Read the the nodes specified under MSFT_lod extension ids and save as children node
+        if (Value *msftLodExt = FindObject(*curExtensions, "MSFT_lod")) {
+            if (Value *lodIds = FindArray(*msftLodExt, "ids")) {
+                for (unsigned int i = 0; i < lodIds->Size(); ++i) {
+                    Value &child = (*lodIds)[i];
+                    if (child.IsUint()) {
+                        // get/create the lod id as child node
+                        Ref<Node> chn = r.nodes.Retrieve(child.GetUint());
```

Comment:
Nit: Comment is a little unclear what is happening. Does this only get an existing node? Or is it creating a node if one doesn't exist?

The `chn` variable name is also unclear. Does this stand for child node?

(Also, looks like `Retrieve` may also throw an `DeadlyImportError` exception)

Reply:
I just followed the naming convention used in assimp ... it used similar names when creating children for the node.
e.g. https://github.com/Esri/assimp/blob/4859e3971f9a69e87f05cb3e8a06112f4e1bac6d/code/AssetLib/glTF/glTFAsset.inl#L1026